.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ["plugin:vue/essential", "@vue/prettier"],
```

Comment:
I'm ok with removing Prettier, but for the record this might be an environment problem - I've been using it successfully within Sublime.